1-mandatory/3-slideshow/slideshow.js

```diff
@@ -1 +1,85 @@
-// Write your code here
+// Write your code here
+// challenge 1
+const flowerArray = [
+  "example-screenshots/1.flower1.jpg",
+  "example-screenshots/2.flower2.jpg",
+  "example-screenshots/3.flower3.jpg",
+];
+
+//  select the btns
+const btnNext = document.getElementById("btn-next");
+console.log(btnNext);
+const btnPrev = document.getElementById("btn-prev");
+console.log(btnPrev);
+
+//  select where the output will be shown
+
+const image = document.querySelector("img");
+
+/* NEXT btn */
+// addevent listerner
+let counter = 0;
+
+btnNext.addEventListener("click", nextPhoto);
+function nextPhoto() {
+  counter++;
+  if (counter >= flowerArray.length) {
+    counter = 0;
+    image.src = flowerArray[counter];
+  } else {
+    image.src = flowerArray[counter];
+  }
+}
+
+/* prevous btn */
+// addevent listerner
+
+btnPrev.addEventListener("click", prevPhoto);
+function prevPhoto() {
+  counter++;
+  if (counter >= flowerArray.length) {
+    counter = 0;
+    image.src = flowerArray[counter];
+  } else {
+    image.src = flowerArray[counter];
+  }
+}

```

Comment:
here you are using the same functionality as in your `nextPhoto`- as this is for the previous photo, you would instead need to be doing `counter--` and your if statement would need to change